### DISCUSSION TITLE
Ubuntu repository specification

### DISCUSSION THREAD

**Discussion by KKLagoda** [Jelmerro/Vieb#460] [Q&A] [answered]
Dear all,

first off, thanks to the dev for this awesome browser, truly enjoy it.

I recently have gotten a PC for work and now need to use ubuntu 20.0.4 instead of arch. Wanted to add the jelmerro.nl repo, but somehow I cannot make it work. The command specified on the downloads page does not work for me (also tried variations) as it tells me it is missing a Release file in the repo. I manually created a jelmerro-vieb.list under /etc/apt/sources.list.d/ and populated it with "deb [arch=amd64,trusted=yes] https://jelmerro.nl/debs/ focal main", yet it is unhappy about the damn release file.

Appreciate any help on the matter, hopefully we can update the instructions on the website.

**Comment by Jelmerro** [comment]
I don't have a release file that I know of, so the `focal main` part should just be left out, as it's the same `.deb` for all systems. Otherwise I don't know that's going wrong, as the commands worked when I tested them in a fresh VM on the latest Mint and Ubuntu.

**Comment by Jelmerro** [comment, marked as answer]
Apparently it's now broken in mint as well, since they now based the new release on the new ubuntu, for which it no longer works. I tried getting it to work, but it keeps refusing the release file no matter which tool I use (I tried aptly, tutorials, manually generating it, nothing worked). Hence I'm dropping it since there is no longer any function or use for it if it doesn't work.

**Reply by KKLagoda** [reply]
Sad to hear that, one more reason for me to go to Arch. Anyways, thanks for investigating it!
